code4sac/trash-ai
# Adding schema files to download

## Changes
Adds the following schema files to the download

- image_schema.json
- image_schema.md
- summary_schema.json
- summary_schema.md

## Purpose
#124 describes the need to add schema files alongside image files when downloading zip files.  This adds the schema files to the download.

## Approach
Adds statics schema files which are downloaded and added to the zip when the download is request.

## Testing
Tested manually locally

1. Pull in the changes to your local copy of this branch and restart Docker.
2. Upload images or use the samples
3. Download the zip and see the schema folder
